e2e/test/scenarios/visualizations-tabular/table.cy.spec.js

```diff
@@ -186,7 +186,7 @@ describe("scenarios > visualizations > table", () => {
           // semantic type
           cy.contains("No special type");
           // fingerprint
-          cy.findByText(/-0\d:00/);
+          cy.findByText("US/Pacific");
```

Comment:
This is the only change necessary for the backport to pass tests. It is not in the master, because test was modified there recently.